pages/my-plants.js

```diff
@@ -0,0 +1,33 @@
+import PlantCard from "@/components/PlantCard";
+import { PlantList } from ".";
+
+export default function MyPlants({ ownedPlants, plants, onToggleOwned }) {
+  const myPlants = plants.filter((plant) =>
+    ownedPlants.find((id) => id === plant.id)
+  );
+
+  if (myPlants.length === 0) {
+    return <p>You don't own any plants yet!</p>;
+  }
+  return (
+    <>
+      <h1>Welcome to your Plants!</h1>
+      <PlantList>
+        {" "}
```

Comment:
This is sometimes added by Visual Studio Code automatically. If it is not useful (like here) you should remove it.